tensorflow/core/ops/nn_ops.cc

```diff
@@ -2412,6 +2412,51 @@ max_output: The float value that the highest quantized output value represents.
 
 )doc");
 
+// This interface is similar to the QuantizedConv2D op's interface:
+// See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/nn_ops.cc#L2369
+//
+// Some discussions for the output_shape parameter:
+// https://github.com/tensorflow/tensorflow/issues/2118
+REGISTER_OP("QuantizedConv2DBackpropInput")
+  .Input("value: Tinput")
+  .Input("filter: Tfilter")
+  .Input("output_sizes: int32")
+  .Input("min_input: float")
+  .Input("max_input: float")
+  .Input("min_filter: float")
+  .Input("max_filter: float")
+  .Output("output: out_type")
+  .Output("min_output: float")
+  .Output("max_output: float")
+  .Attr("Tinput: quantizedtype")
+  .Attr("Tfilter: quantizedtype")
+  .Attr("out_type: quantizedtype = DT_QINT32") // default quantized type
+  // .Attr("Tinput: numbertype")
+  // .Attr("Tfilter: numbertype")
+  // .Attr("out_type: numbertype")
+  .Attr("strides: list(int)")
+  .Attr(GetPaddingAttrString()) // add padding string
+  .SetShapeFn([] (InferenceContext *c) {
+    // TODO: Don't understand this shape handle funtion for now, will check later
```

Comment:
Please copy the code from the Conv2DBackpropInput shape function here:
https://github.com/kumasento/tensorflow/blob/3d50e2ef0f823b3c2e7dc5070734fe418d4ce32b/tensorflow/core/ops/nn_ops.cc#L566